client/blocks/image-editor/test/image-editor-toolbar.jsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import { useSandbox } from 'test/helpers/use-sinon';
+
+describe( 'ImageEditorToolbar', () => {
+	useFakeDom();
+
+	let ImageEditorToolbar,
+		defaultProps,
+		wrapper;
+
+	useSandbox( ( sandbox ) => {
+		defaultProps = {
+			translate: sandbox.stub().returns( 'Hajimemashite!' ),
```

Comment:
romanji 😆 

Similar to the other test, we can either pass through the real translate, or use lodash identity.

eg 
```jsx
import { translate } from 'i18n-calypso';
//...
wrapper = shallow( <ImageEditorToolbar { ...defaultProps } translate={} /> );
```